hw/top_earlgrey/lint/top_earlgrey_lint_cfgs.hjson

```diff
@@ -290,7 +290,7 @@
              {    name: top_earlgrey
                   fusesoc_core: lowrisc:systems:top_earlgrey
                   import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
-                  rel_path: "hw/top_earlgrey/lint/{tool}"
+                  rel_path: "hw/top_earlgrey/chiplevel/top_earlgrey/lint/{tool}"
```

Comment:
Was the existing path not already unique?

Reply:
Not entirely - the `top_earlgrey` summary report was written to the same location as the `top_earlgrey` result before.

Reply:
Right - but they were separate html files (summary.html v/s result.html). Is that no longer enough?

Reply:
The problem I ran into is that the summary report gets written after the individual `result.html` report. Hence, the odir logic above detects the `latest` folder containing results.html and renames it to `latest_<ts>` before writing the summary. Further disambiguation would solve that issue.